.test-infra/jenkins/JobBuilder.groovy

```diff
@@ -47,7 +47,7 @@ class JobBuilder {
 
   private void defineAutoPostCommitJob(name) {
```

Comment:
Since you create PrecommitBuilder, it is best to rename this class to PostcommitBuilder.
I would also rename both classes to "xxxxJobBuilder"